internal/pkg/tekton/pipeline_run_builder.go

```diff
@@ -127,12 +156,12 @@ func NewPipelineRunBuilder(name, namespace string) *PipelineRunBuilder {
 											},
 											ComputeResources: corev1.ResourceRequirements{
 												Requests: corev1.ResourceList{
-													"cpu":    resource.MustParse("150m"),
-													"memory": resource.MustParse("512Mi"),
+													"cpu":    resource.MustParse("300m"),
+													"memory": resource.MustParse("1Gi"),
```

Comment:
Is increasing `requests.limits` sufficient for us? And since the limit of cpu is still `300m`, I don't think it makes sense to increase `cpu` request to `300m`.

Reply:
How about `requests.memory`? Is original `512Mi` sufficient as the minimal requirement for pod to start?